tests/integration/test_governed_evolution_runtime_smoke.py

```
@@ -0,0 +1,99 @@
+            self.skipTest("PowerShell executable not available in PATH")
+
+        self.assertTrue(SMOKE_SCRIPT.exists(), f"Missing smoke script: {SMOKE_SCRIPT}")
+
+        outputs_root = REPO_ROOT / "outputs"
+        outputs_root.mkdir(parents=True, exist_ok=True)
+        artifact_root = outputs_root / f"pytest-governed-evolution-{uuid.uuid4().hex}"
+        artifact_root.mkdir(parents=True, exist_ok=True)
+        try:
+            run_id = "pytest-governed-evolution-smoke"
+            try:
+                completed = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)